spyql/processor.py

```diff
@@ -28,6 +28,16 @@ def init_vars():
         {},
         vars,
     )
+
+    try:
+        # user defined imports, functions, etc
+        with open(os.path.expanduser("~/.spyql.py")) as f:
```

Comment:
For a long time there has been a debate on where should applications look for / write their data. There are plenty of applications that are all over the place but there's actually an attempt to standardize the directory structure. Have a look at [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html). 
I think that a better path for this file would be `$HOME/.config/sqyql/config.py`

In case you're curious about what applications actually comply with the standard, the arch linux community maintains a [list of applications] with different compliance levels.

Reply:
Makes sense! I was not aware, nice catch.